server/mocks/5000-kio.js

```diff
@@ -160,6 +160,11 @@ server.put('/apps/:id/versions/:ver', function(req, res) {
             versions[id][ver] = req.body;
             versions[id][ver].application_id = id;
             versions[id][ver].id = ver;
+            // delete approvals for this version
+            if (approvals[id] && approvals[id][ver]) {
+                console.log('deleting approvals for', id, ver);
```

Comment:
Not really, was for debugging.

----- Ursprüngliche Nachricht -----
Von: "Emanuel Lauria" <notifications@github.com>
Gesendet: ‎19.‎05.‎2015 18:04
An: "zalando-stups/yourturn" <yourturn@noreply.github.com>
Cc: "Nikolaus Piccolotto" <prayerslayer@gmail.com>
Betreff: Re: [yourturn] Update approvals (#103)

In server/mocks/5000-kio.js:
> @@ -160,6 +160,11 @@ server.put('/apps/:id/versions/:ver', function(req, res) {
>              versions[id][ver] = req.body;
>              versions[id][ver].application_id = id;
>              versions[id][ver].id = ver;
> +            // delete approvals for this version
> +            if (approvals[id] && approvals[id][ver]) {
> +                console.log('deleting approvals for', id, ver);
do we need this console.log?
—
Reply to this email directly or view it on GitHub.